config/locales/fi.yml

```diff
@@ -0,0 +1,46 @@
+fi:
```

Comment:
Well, I can't speak Finnish, so I cannot really review this :slightly_smiling_face: I would use DeepL or Google Translate to sanity-check the translations.

Reply:
Did that using Google Translate and also sent a screenshot of the translated page to Isa to have a look at! :)